tooling/build-types-and-manifests.ts

```diff
@@ -49,6 +49,43 @@ addManifest('firefox-consumer', manifest => {
   delete manifest.minimum_chrome_version;
 });
 
+addManifest('thunderbird-consumer', manifest => {
```

Comment:
are there many differences with `firefox-consumer` manifest?
probably it'll be simpler to reuse manifest generated for firefox and just update fields which are different

Reply:
Thank you, Roma - I missed that. I have update the code to reuse a specific manifest file and update the fields that differs. thanks!